### ISSUE TITLE
Document data upload/backup workflow

### ISSUE THREAD

**Issue by leewujung** [uw-echospace/union-bay-bats#102]
A reminder to document data upload/backup workflow, including the following:
- general workflow
- OSN setup and resources
  - [rclone configuration](https://openstoragenetwork.readthedocs.io/en/latest/accessing-datasets.html#rclone-configuration)

**Comment by leewujung** [comment]
I assigned this to myself before, and I'll continue to backup to the OSN bucket since I am the only one with write permission.

But @ItsUnagii I think this is a good opportunity for you to put together a documentation for the data download flow from the SD card and run detection in command line as you learn the steps and take over the lead for fieldwork.

This is related to #153.

**Comment by ItsUnagii** [comment]
I believe the SD data download is already documented in the readme md, is there anything specifically you want me to do with it?
Also, the command line detection stuff should be outlined in the batdetect repository - I think I will summarize the relevant information into an md file and put it in here.
I'll try the OSN setup sometime this week.

**Comment by leewujung** [comment]
> I believe the SD data download is already documented in the readme md, is there anything specifically you want me to do with it?

I would recommend review the current README and update it for any missing pieces - READMEs can always be improved!

> Also, the command line detection stuff should be outlined in the batdetect repository - I think I will summarize the relevant information into an md file and put it in here.

Sounds good!

> I'll try the OSN setup sometime this week.

👍

**Comment by ItsUnagii** [comment]
Drafting command line detecting documentation right now. I'm thinking about just making a branch, and then sending it back to you to review before merging.

In the SD data download, it says "check out with yjeong" at the bottom in the "Tidy Up" section. I didn't see anyone similar to that name in Slack, so I was wondering if we still needed to keep that. Also, are we still checking in with you every field deployment?

I think something that might be helpful in the SD card download is what all the flags for the rsync command do. I was curious about this and asked Aditya about it the other day, and the two of us had to google each flag.

**Comment by leewujung** [comment]
> Drafting command line detecting documentation right now. I'm thinking about just making a branch, and then sending it back to you to review before merging.

Yep - please make a fork under your own account and then make a branch there, and then PR to this upstream repo. Request reviews from both me and Aditya.

> In the SD data download, it says "check out with yjeong" at the bottom in the "Tidy Up" section. I didn't see anyone similar to that name in Slack, so I was wondering if we still needed to keep that. Also, are we still checking in with you every field deployment?

I think that was a typo, and no need to check with YeonJoon on this. Check in with Aditya instead!

> I think something that might be helpful in the SD card download is what all the flags for the rsync command do. I was curious about this and asked Aditya about it the other day, and the two of us had to google each flag.

Great that you did that - I would recommend including those into the doc!
